tox.ini

```diff
@@ -4,9 +4,25 @@ envlist = py27
 [testenv]
 deps = pytest
 install_command = pip install -e .[test] --find-links {toxinidir}/3rdparty {opts} {packages}
-# TODO(xujyan): Fix up 'test_scheduler_runs'.
 commands = py.test -v -k 'not test_scheduler_runs' tests {posargs:}
 
+[testenv:pex]
+deps =
+  pex
```

Comment:
this should be 'pex==0.9.0' or 'pex<1'.  pex 1.0 is going to be published soon and the CLI arguments will change.